tools/jslint/check-all.js

```diff
@@ -60,7 +60,7 @@ var globals =
     + ' viewport process_loaded_for_unread'
     + ' have_scrolled_away_from_top'
     + ' maybe_scroll_to_selected recenter_pointer_on_display suppress_scroll_pointer_update'
-    + ' fast_forward_pointer recent_subjects unread_subjects'
+    + ' fast_forward_pointer recent_subjects unread_subjects recent_private_messages'
```

Comment:
I think it's message_store.js, not stream_list.js for that one. Since it's a 'twin' to recent_subjects, would you like me to align them & move both of them to the exports within message_store and then reference them everywhere as recent_subjects and message_store.recent_private_messages